task-sdk/src/airflow/sdk/definitions/asset/__init__.py

```diff
@@ -60,7 +60,7 @@
 
 
 @attrs.define(frozen=True)
-class AssetUniqueKey:
+class AssetUniqueKey(attrs.AttrsInstance):
```

Comment:
What this base class for?

Reply:
To convince Mypy we can do `attrs.asdict` to an instance. It does nothing at runtime.